### PR TITLE
php: 7.1.22 -> 7.1.23, 7.2.10 -> 7.2.11

### PR DESCRIPTION
**Pull request by etu**
###### Motivation for this change
Let's hope that they have fixed the macOS build this time around. Otherwise I'll roll back macOS again to at least keep the linux users up-to-date.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

cc @xeji

### CODE REVIEW

**Comment by etu** [comment]
And it wasn't fixed on darwin. Pushed new versions for Linux.

**Comment by etu** [comment]
Ok, the old versions worked on darwin. Let's try the new versions.

@GrahamcOfBorg build php71 php72

File: pkgs/development/interpreters/php/default.nix [hunk 1]
```diff
@@ -227,31 +230,15 @@ in {
...
-        sha256 = "1rky321gcvjm0npbfd4bznh36an0y14viqcvn4yzy3x643sni00z";
-      }
-    else
-      {
-        version = "7.2.10";
-        sha256 = "17fsvdi6ihjghjsz9kk2li2rwrknm2ccb6ys0xmn789116d15dh1";
-      }
-  );
+  php72 = generic {
+    version = "7.2.8";
```
Comment (xeji) [inline pkgs/development/interpreters/php/default.nix]: Why downgrade everything to 7.2.8 ? Or are you still testing?

Comment (etu) [reply pkgs/development/interpreters/php/default.nix]: Yeah, that's kinda why I went to bed I think :smile: 

I will pick this up again tonight (about 10-11 hours from now).

**Comment by etu** [comment]
@GrahamcOfBorg build php71 72

**Comment by etu** [comment]
@xeji I've given up on darwin for this time. But it's been reported to upstream that it's still broken. And they managed to catch the logs this time before it was gone from of-borg log hosting.

So I've removed the darwin WIP commits and the current ones only affect Linux.

@GrahamcOfBorg build php71 php72

**Pull request merged**